libs/ledger-live-common/src/families/bitcoin/wallet-btc/explorer/index.ts

```diff
@@ -48,10 +47,10 @@ class BitcoinLikeExplorer implements IExplorer {
       });
     }
 
-    const client = axios.create(clientParams);
-    this.underlyingClient = client;
+    this.client = axios.create(clientParams);
```

Comment:
You can try removing these custom params.

Reply:
fixed